### DISCUSSION TITLE
Why does does my deployed app get health checks 2-3 times per second?

### DISCUSSION THREAD

**Discussion by jeromedalbert** [basecamp/kamal#811] [Q&A] [answered]
I deployed a dummy Rails app on a single VPS with Kamal and I am getting a lot of health checks to `/up` in the logs, like 2-3 per second:

![Screenshot 2024-05-18 at 10 09 48 AM](https://github.com/basecamp/kamal/assets/236461/05ddae4a-1652-4b51-9e18-6f2e67b0ca1e)

Why are these health checks running that often? And what is triggering them, Docker or Traefik? Maybe the check intervals can be configured somewhere.

**Comment by acidtib** [comment, marked as answer]
https://kamal-deploy.org/docs/configuration/healthchecks/

**Reply by jeromedalbert** [reply]
```yml
healthcheck:
  interval: 20s
```

That works, thanks. I wrongly thought those healthchecks were for `kamal deploy`.

However I had to revert that solution, as changing the interval from the default 1s to 20s makes my deploys fail when kamal checks for healthy instances that should become unhealthy, and my deploys felt more sluggish too. 

So although this works, it also causes more issues down the line, so it is currently not practical for me.
